tests/sqlparser_databricks.rs

```diff
@@ -28,3 +29,98 @@ fn test_databricks_identifiers() {
         SelectItem::UnnamedExpr(Expr::Value(Value::DoubleQuotedString("Ä".to_owned())))
     );
 }
+
+#[test]
+fn test_databricks_exists() {
+    // exists is a function in databricks
+    assert_eq!(
+        databricks().verified_expr("exists(array(1, 2, 3), x -> x IS NULL)"),
+        call(
+            "exists",
+            [
+                call(
+                    "array",
+                    [
+                        Expr::Value(number("1")),
+                        Expr::Value(number("2")),
+                        Expr::Value(number("3"))
+                    ]
+                ),
+                Expr::Lambda(LambdaFunction {
+                    params: OneOrManyWithParens::One(Ident::new("x")),
+                    body: Box::new(Expr::IsNull(Box::new(Expr::Identifier(Ident::new("x")))))
+                })
+            ]
+        ),
+    );
+
+    let res = databricks().parse_sql_statements("SELECT EXISTS (");
+    assert_eq!(
+        // TODO: improve this error message...
```

Comment:
was the todo meant for this PR or as a general improvement/followup with error reporting?

Reply:
the latter